### DISCUSSION TITLE
Project Dependencies

### DISCUSSION THREAD

**Discussion by ghost** [clementwzk/OpenCalc#162] [Q&A]
I am beginner in open source so need some guidance...

There are some new versions of dependencies available in this project so should I update all the dependencies and create a PR or should I work on the project without updating......

**Comment by clementwzk** [comment]
Do a PR only for dependencies then another if you want to work on a feature or something like that

**Reply by ghost** [reply]
Thanks for reply....

**Comment by inson1** [comment]
What about closing this? I dont think, its worth it adding this to Most Asked Questions or Contributing guidelines, because its really beginner question.
